hazelcast-spring/src/test/resources/com/hazelcast/spring/fullConfig-applicationContext-hazelcast.xml

```diff
@@ -49,8 +49,8 @@
                 <hz:property name="hazelcast.merge.first.run.delay.seconds">5</hz:property>
                 <hz:property name="hazelcast.merge.next.run.delay.seconds">5</hz:property>
             </hz:properties>
-            <hz:wan-replication name="testWan" snapshot-enabled="false">
-                <hz:target-cluster group-name="tokyo" group-password="tokyo-pass">
+            <hz:wan-replication name="testWan">
+                <hz:target-cluster group-name="tokyo" group-password="tokyo-pass" snapshot-enabled="false">
```

Comment:
Can you please ensure that this is mentioned in the Release Notes? That requires XML config change when upgraded to 3.7.

please also fail fast when snapshot-enabled=true but Batch Publisher is not used.